bolt/include/bolt/Core/BinaryContext.h

```diff
@@ -377,6 +381,17 @@ class BinaryContext {
     RtLibrary = std::move(Lib);
   }
 
+  const MCPseudoProbeDecoder *getPseudoProbeDecoder() const {
+    return PseudoProbeDecoder.get();
+  }
+
+  MCPseudoProbeDecoder &
+  setPseudoProbeDecoder(std::unique_ptr<MCPseudoProbeDecoder> Decoder) {
```

Comment:
nit: it's a bit unconventional to have setter return a reference. Nearby code all have a setter with void return and a dedicated setter. Suggest to be consistent.